custom_components/tuya_local/devices/etop_fcu_thermostat.yaml

```diff
@@ -0,0 +1,133 @@
+name: FCU thermostat
+products:
+  - id: psetc59fvtm5gexl
+    name: ETOP-FCU(CF)
+    name: Jaga JRT-100TW
```

Comment:
There can only be one name per id, so these need to be combined on one line if it is confirmed both models have the same product id.

Reply:
Done.